src/main/java/hudson/plugins/doxygen/DoxygenBuilder.java

```diff
@@ -95,70 +97,62 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         try {
             if (!doxygenPath.exists()) {
                 listener.fatalError(String.format("The path to Doxygen executable doesn't exist : \"%s\"", doxygenPath));
-                return false;
             }
         } catch (IOException e) {
             listener.fatalError(String.format("Failed checking the existence of Doxygen executable : \"%s\"", doxygenPath));
             Util.displayIOException(e, listener);
-            return false;
+            return;
         }
-        
+
         // Raise an error if the doxyfile isn't set
         String pathToDoxyfile = getDoxyfilePath();
         if (pathToDoxyfile == null || pathToDoxyfile.trim().length() == 0)
         {
             listener.fatalError("The Doxyfile path is empty. Please review the build step configuration.");

```

Comment:
`AbortException` (though the original impl apparently did *not* treat this as fatal)